.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Build and Publish Images
+
+on:
+  push:
+    branches:
+      - redhat-3.5
+
+jobs:
+  quay-image:
+    name: Calculate Quay Image Digest
+    runs-on: 'ubuntu-latest'
+    outputs:
+      digest: ${{ steps.set-output.outputs.digest }}
+    env:
+      IMAGE_REGISTRY: quay.io/projectquay
+      TAG: 3.5
+    steps:
+      - name: Pull Image
+        id: pull-image
+        run: docker pull "${IMAGE_REGISTRY}"/quay:"${TAG}"
+      - name: Set Output
+        id: set-output
+        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/quay:${TAG})"
+
+  clair-image:
+    name: Calculate Clair Image Digest
+    runs-on: 'ubuntu-latest'
+    outputs:
+      digest: ${{ steps.set-output.outputs.digest }}
+    env:
+      IMAGE_REGISTRY: quay.io/projectquay
+      TAG: 3.5
+    steps:
+      - name: Pull Image
+        id: pull-image
+        run: docker pull "${IMAGE_REGISTRY}"/clair:"${TAG}"
+      - name: Set Output
+        id: set-output
+        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/clair:${TAG})"
+
+  builder-image:
+    name: Calculate Builder Image Digest
+    runs-on: 'ubuntu-latest'
+    outputs:
+      digest: ${{ steps.set-output.outputs.digest }}
+    env:
+      IMAGE_REGISTRY: quay.io/projectquay
+      TAG: 3.5
+    steps:
+      - name: Pull Image
+        id: pull-image
+        run: docker pull "${IMAGE_REGISTRY}"/quay-builder:"${TAG}"
+      - name: Set Output
+        id: set-output
+        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/quay-builder:${TAG})"
+
+  qemu-builder-image:
+    name: Calculate Qemu Builder Image Digest
+    runs-on: 'ubuntu-latest'
+    outputs:
+      digest: ${{ steps.set-output.outputs.digest }}
+    env:
+      IMAGE_REGISTRY: quay.io/projectquay
+    steps:
+      - name: Pull Image
+        id: pull-image
+        run: docker pull "${IMAGE_REGISTRY}"/quay-builder-qemu:latest
+      - name: Set Output
+        id: set-output
+        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/quay-builder-qemu:latest)"
+
+  operator-image:
+    name: Publish 3.5 Operator Image
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          ref: 'redhat-3.5' # must use correct version branch!
```

Comment:
This seems redundant with 
```
on:
  push:
    branches:
      - redhat-3.5
```
Not a deal breaker but seems odd we have to enforce this.

Reply:
Talked about this IRL - resolving the conversation